core/src/main/kotlin/org/evomaster/core/search/service/mutator/StandardMutator.kt

```diff
@@ -181,7 +183,7 @@ class StandardMutator<T> : Mutator<T>() where T : Individual {
 
     private fun handleDoubleGene(gene: DoubleGene) {
         //TODO min/max for Double
-
+        val d = gene.value
```

Comment:
why this variable?

Reply:
thanks. it is a mistake. when i implemented archive-based mutation, i edit a bit. but i forgot to change it.